python/pyarrow/tests/test_dataset.py

```diff
@@ -1117,6 +1115,15 @@ def test_open_dataset_from_uri_s3(s3_connection, s3_server):
     with fs.open_output_stream("mybucket/data.parquet") as out:
         pq.write_table(table, out)
 
+    return table, path, fs, uri, host, port, access_key, secret_key
```

Comment:
Since I only unpack it below (like `table, path, _, _, host, port, access_key, secret_key = s3_example_simple`), I am not sure using a namedtuple will help much (the unpacking won't check names or so)

Reply:
Well, it helps avoid the unpacking.

Reply:
I would personally prefer to keep the unpacking, otherwise that will make the rest of the code IMO less readable.

To ensure we are talking about the same, you would propose to do like this?

```
...
dataset = ds.dataset(s3_example_simple.path, format="parquet", filesystem=s3_example_simple.fs)
...
```

(or with a shorter alias for ``s3_example_simple``)

Reply:
Ok, fair enough, let's keep it like this.